account_payment_mode/migrations/18.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2025 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env, "account_payment_mode", "18.0.1.0.0/noupdate_changes.xml"
```

Comment:
```suggestion
        env, "account_payment_mode", "migrations/18.0.1.0.0/noupdate_changes.xml"
```

I had an error that seems to be fixed this way

Reply:
Fixed, thanks